Source/UINavigation/Private/UINavInputBox.cpp

```diff
@@ -119,8 +119,6 @@ void UUINavInputBox::CreateInputKeyWidgets()
 	TArray<FInputActionKeyMapping> Actions;
 	TArray<FInputAxisKeyMapping> Axes;
 
-	ProcessInputName();
```

Comment:
I'm pretty sure this still needs to be called.

Reply:
It is called in this function:

```
void UUINavInputBox::CreateKeyWidgets()
{
	InputButtons = { InputButton1, InputButton2, InputButton3 };
	**ProcessInputName();**

	if (IS_ENHANCED_INPUT)
	{
		CreateEnhancedInputKeyWidgets();
	}
	else
	{
		CreateInputKeyWidgets();
	}
}

```

Then it is called again inside CreateInputKeyWidgets(); which is called by that function. I removed it because it is being called twice.